README.md

```diff
@@ -35,64 +33,51 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 
 * OS X
-    * [xhyve driver](./docs/docker-machine-drivers.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion) installation
+    * [xhyve driver](./docs/docker-machine-drivers.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
 * Linux
-    * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [KVM](http://www.linux-kvm.org/) installation
+    * [KVM](./docs/docker-machine-drivers.md#kvm-driver) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 * Windows
-    * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [Hyper-V](https://technet.microsoft.com/en-us/library/mt169373.aspx) installation
+    * [Hyper-V](https://technet.microsoft.com/en-us/library/mt169373.aspx) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-The driver can be selected via the `--vm-driver=xxx` flag of `minishift start`. See
-[docker machine drivers](./docs/docker-machine-drivers.md) for more details on supported drivers
-and how to install them.
+Minishift embeds VirtualBox and VMware Fusion drivers so there are no additional configurations required to use them. However, some of the drivers, currently, require a few extra steps, see [docker machine drivers](./docs/docker-machine-drivers.md) for more details on supported drivers and how to install them.
 
-**Note**: For most hyervisor VT-x/AMD-v virtualization must be enabled in BIOS. For Hyper-V, however,
-it needs to be disabled.
-
-<a name="instructions"></a>
-### Instructions
+The driver can be selected via the `--vm-driver=xxx` flag of `minishift start` as mentioned in the Quickstart section below. See the [Troubleshooting](/docs/troubleshooting.md#kvm-drivers) guide, in case you encounter any issues with the drivers.
 
-Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. Then copy the contained binary to your preferred
-location and optionally ensure it is added to your _PATH_.
+**Note**: For most hypervisors, VT-x/AMD-v virtualization must be enabled in the BIOS. For Hyper-V, however,
+it needs to be disabled.
 
-<a name="macos"></a>
-#### macOS
+<a name="quickstart"></a>
+## Quickstart
 
-##### Stable
+1. Download the archive matching your host OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it. For example, for OS X you would need to download [minishift-1.0.0-beta.1-darwin-amd64.tgz](https://github.com/minishift/minishift/releases/download/v1.0.0-beta.1/minishift-1.0.0-beta.1-darwin-amd64.tgz).
 
-On macOS you can also use [Homebrew Cask](https://caskroom.github.io) to install Minishift:
+ **Note:** On macOS you can also use [Homebrew Cask](https://caskroom.github.io) to install the stable version of Minishift:
```

Comment:
That's a pretty big side note. I think we should take this hole Homebrew stuff and create another subsection '###Homebrew' were we describe this option, instead of munching it into the middle of the default instructions.